<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kara Kaplı Defter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="simgeler.qrc">
    <normaloff>:/new/prefix1/icons/kkd64.png</normaloff>:/new/prefix1/icons/kkd64.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>9</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget">
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout">
           <property name="spacing">
            <number>11</number>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="buttonClear">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Arama alanını ve sonuçları temizler</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="simgeler.qrc">
               <normaloff>:/new/prefix1/icons/locationbar_erase.png</normaloff>:/new/prefix1/icons/locationbar_erase.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&amp;Ara:</string>
             </property>
             <property name="buddy">
              <cstring>lineAra</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineAra">
             <property name="frame">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboAra">
             <item>
              <property name="text">
               <string>Müşteriler</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Siparişler</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>9</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Sonuçlar</string>
             </property>
             <property name="buddy">
              <cstring>treeSonuclar</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeSonuclar">
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <column>
              <property name="text">
               <string>Tür</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Açıklama</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Detaylı Bilgiler</string>
          </property>
          <property name="buddy">
           <cstring>textDetay</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textDetay">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSe_enekler">
    <property name="title">
     <string>Seçenekler</string>
    </property>
    <addaction name="actionAyarlar"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuKay_t">
    <property name="title">
     <string>Kayıt</string>
    </property>
    <addaction name="actionNewContact"/>
    <addaction name="actionNewOrder"/>
    <addaction name="separator"/>
    <addaction name="actionDaily"/>
    <addaction name="separator"/>
    <addaction name="actionArama"/>
    <addaction name="separator"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDel"/>
   </widget>
   <widget class="QMenu" name="menuHakk_nda">
    <property name="title">
     <string>Hakkında</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSe_enekler"/>
   <addaction name="menuKay_t"/>
   <addaction name="menuHakk_nda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewContact"/>
   <addaction name="actionNewOrder"/>
   <addaction name="separator"/>
   <addaction name="actionEdit"/>
   <addaction name="separator"/>
   <addaction name="actionArama"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
  </widget>
  <action name="actionNewContact">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/identity.png</normaloff>:/new/prefix1/icons/identity.png</iconset>
   </property>
   <property name="text">
    <string>Yeni Müşteri</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionNewOrder">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/kontact_todo.png</normaloff>:/new/prefix1/icons/kontact_todo.png</iconset>
   </property>
   <property name="text">
    <string>Yeni Sipariş</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAyarlar">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/configure.png</normaloff>:/new/prefix1/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Ayarlar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/edit.png</normaloff>:/new/prefix1/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Düzenle</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/editdelete.png</normaloff>:/new/prefix1/icons/editdelete.png</iconset>
   </property>
   <property name="text">
    <string>İptal Et</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/fileprint.png</normaloff>:/new/prefix1/icons/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Yazdır</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/exit.png</normaloff>:/new/prefix1/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
  <action name="actionArama">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/filefind.png</normaloff>:/new/prefix1/icons/filefind.png</iconset>
   </property>
   <property name="text">
    <string>Arama</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/kkd64.png</normaloff>:/new/prefix1/icons/kkd64.png</iconset>
   </property>
   <property name="text">
    <string>Hakkında</string>
   </property>
  </action>
  <action name="actionDaily">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/date.png</normaloff>:/new/prefix1/icons/date.png</iconset>
   </property>
   <property name="text">
    <string>Rapor</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="simgeler.qrc">
     <normaloff>:/new/prefix1/icons/help.png</normaloff>:/new/prefix1/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Yardım</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="simgeler.qrc"/>
 </resources>
 <connections/>
</ui>
